src/components/daydream-controls.js

```diff
@@ -83,7 +81,14 @@ module.exports.Component = registerComponent('daydream-controls', {
   },
 
   checkIfControllerPresent: function () {
-    var isPresent = this.isControllerPresent(this.el.sceneEl, GAMEPAD_ID_PREFIX, {hand: this.data.hand});
+    var handedness = this.data.hand === '' ? ['left', 'right'] : [this.data.hand];
```

Comment:
you can take a look at latest gearvr-controls PR changes, which work even for zero DOF with no hand if you specify that those controllers should be allowed (and so should also work for 3DOF that has no hand)